packages/ethereum-contracts/scripts/gov-create-new-app-registration-key.js

```diff
@@ -54,8 +73,12 @@ module.exports = eval(`(${S.toString()})({
         )
     );
     console.log("App key", appKey);
+    console.log("Expiration date", new Date(expirationTs * 1000)); // print human readable
 
+    // Note that we are NOT using gov.whiteListNewApp here, because it doesn't support setting
+    // an expiration date and will eventually be deprecated.
+    // Instead we use the lower level setConfig directly.
     await sendGovernanceAction(sf, (gov) =>
-        gov.whiteListNewApp(sf.host.address, appKey)
+        gov.setConfig(sf.host.address, ZERO_ADDRESS, appKey, expirationTs)
```

Comment:
what do we do with the `host.whiteListNewApp`?

Reply:
>  The convenience function gov.whiteListNewApp isn't used, should be deprecated / removed on next gov contract update. We could instead change and add a param for expiration date, but not really needed (we can keep using the generic setConfig instead).

Reply:
we could already remove it, and next scheduled update will get it removed.

Reply:
done